cht-llama-cpp/main.py

```diff
@@ -10,7 +10,7 @@
 
 load_dotenv()
 
-MODEL_PATH = f"./ml/models/{os.getenv('MODEL_ID', 'mistral-7b-instruct-v0.1.Q5_0')}.gguf"
+MODEL_PATH = f"./ml/models/{os.getenv('MODEL_ID', 'yarn-mistral-7b-128k.Q4_K_M')}.gguf"
```

Comment:
This means we always have 128k yarn as base model then?

Reply:
when running through registry, we would always be overriding [this](https://github.com/premAI-io/prem-registry/blob/dcbac411276e96ce278d99089a0d94ab502eb9bc/chat-mistral-7b-instruct/manifest.json#L27), but if someone runs the server directly without passing `--model-path` then it'd always look for yarn-mistral